### PR TITLE
RW-186 site slogan

### PR DESCRIPTION
**Pull request by left23**
Screenshots on RW-186

### CODE REVIEW

**Comment by left23** [comment]
There's one question about the `/topics/covid-19` on the ticket. The rest can be reviewed.

File: html/themes/custom/common_design_subtheme/templates/cd/cd-header/cd-header.html.twig [hunk 1]
```diff
@@ -3,6 +3,16 @@
...
     {% include '@common_design_subtheme/cd/cd-header/cd-global-header.html.twig' %}
     {% include '@common_design_subtheme/cd/cd-header/cd-site-header.html.twig' %}
+
+    <div id="site-slogan">
+      <div class="cd-container">
+        <!-- TEMPORARY - COVID-19 - disabled
+        {% trans %}<span>Informing humanitarians worldwide 24/7 &mdash; <span>a service provided by&nbsp;<span class="ocha">UN OCHA</span></span></span>{% endtrans %}
+         -->
+        <!-- TEMPORARY - COVID-19 -->
+        <span><strong>{% trans %}RW COVID-19 page:&nbsp;</strong><a href="/topics/covid-19">Find latest updates on global humanitarian responses{% endtrans %}</a></span>
```
Comment (orakili) [inline html/themes/custom/common_design_subtheme/templates/cd/cd-header/cd-header.html.twig:13]:
```suggestion
        <span><strong>{% trans %}RW COVID-19 page:&nbsp;</strong><a href="/topic/covid-19">Find latest updates on global humanitarian responses{% endtrans %}</a></span>
```

Or the other suggestion below.

Comment (orakili) [inline html/themes/custom/common_design_subtheme/templates/cd/cd-header/cd-header.html.twig:13]:
```suggestion
        {% set covid_topic_url = url('entity.node.canonical', {'node': 3563703})|render %}
        <span><strong>{% trans %}RW COVID-19 page:&nbsp;</strong><a href="{{ covid_topic_url  }}">Find latest updates on global humanitarian responses{% endtrans %}</a></span>
```

Or the other suggestion above

**Review by orakili — DISMISSED** [review]
Looks good, thanks!

Added 2 suggestions for the COVID 19 topic URL. Either is fine.

**Pull request merged**